app/admin/project.rb

```diff
@@ -0,0 +1,62 @@
+ActiveAdmin.register Project do
+  permit_params :name, user_attributes:[:id, :email],user_ids: [:id]
+
+  index do
+    selectable_column
+    id_column
+    column :name
+    actions
+  end
+
+  form do |f|
+    f.input :name
+    f.input :users, as: :check_boxes, multiple: true, member_label: :email
+    f.submit
+  end
+
+  show do
+    panel "Project" do
+      h3 "ID - #{project.id}"
+      h3 "Name - #{project.name}"
+      table_for project.users do
+        column :id
+        column :email
+      end
+    end
+  end
+
+
+  controller do
+    def new
+      @project = Project.new
+    end
+    def create
```

Comment:
Assignment Branch Condition size for create is too high. [18.14/15]<br>Use empty lines between defs.